p2-repository/pom.xml

```diff
@@ -0,0 +1,359 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.eclipse.collections</groupId>
+    <artifactId>eclipse-collections-parent</artifactId>
+    <version>8.2.0-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.eclipse.collections</groupId>
+  <artifactId>p2-repository</artifactId>
+  <version>8.2.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <name>Eclipse Collections p2 Repository Build</name>
+
+  <modules>
+	<module>org.eclipse.collections</module>
+  </modules>
+
+  <properties>
+
+    <!-- include everything from all dependencies by default -->
+    <recipe.includes>**/*</recipe.includes>
+
+    <!-- but filter out maven instructions, we'll generate new ones below -->
+    <recipe.excludes>META-INF/maven/**/*.*</recipe.excludes>
+
+    <!-- BND specific instructions which can be overridden by recipe -->
+    <recipe.Bundle-Name>%bundleName</recipe.Bundle-Name>
+    <recipe.Bundle-Description />
+    <recipe.Bundle-Vendor>%bundleVendor</recipe.Bundle-Vendor>
+
+    <!-- remove some clutter from the generated manifests -->
+    <recipe.removeheaders>Ignore-Package,Include-Resource,Private-Package,Embed-Dependency,Built-By,Build-Jdk,Bnd-*,Tool</recipe.removeheaders>
+
+    <!-- for Orbit, by default we'll also remove the Require-Bundle header; recipe poms may override if they need this header -->
+    <recipe.removeadditionalheaders>Require-Bundle</recipe.removeadditionalheaders>
+
+    <!-- read BND settings from 'osgi.bnd' located next to pom.xml -->
+    <recipe.bndfile>osgi.bnd</recipe.bndfile>
+
+    <!-- pattern/format used for the qualifier -->
+    <recipe.qualifier.format>'v'yyyyMMdd-HHmm</recipe.qualifier.format>
+
+    <!-- project defaults -->
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <!-- versions used for plug-ins below -->
+    <tycho-version>1.0.0</tycho-version>
+    <tycho-extras-version>1.0.0</tycho-extras-version>
+    <maven-resources-version>2.7</maven-resources-version>
+    <maven-dependency-version>2.10</maven-dependency-version>
+
```

Comment:
maven-resources, maven-dependency versions are not necessary. See comments below.